packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -76,9 +76,11 @@ async function requestSignatures(request: Request, response: Response) {
   obs.observe({ entryTypes: ['measure'], buffered: true })
 
   const signers = JSON.parse(config.odisServices.signers) as SignerService[]
+  let timedOut = false
   const signerReqs = signers.map((service) => {
     const controller = new AbortController()
```

Comment:
Wasn't Zviad saying that we should move this outside of the signers.map loop so that we don't create a new controller for each signer?

Reply:
I'm referrring to this line 
`const controller = new AbortController()`